src/utils/resin.ts

```diff
@@ -0,0 +1,28 @@
+const RESIN_PREFIX = 'data-resin-';
+
+export function getResinAttribute(name: string): string {
+    return `${RESIN_PREFIX}${name}`;
+}
+
+export function stringify(value: unknown): string | null {
+    if (typeof value === 'object' || typeof value === 'function') {
```

Comment:
Do we need to handle `Symbol` type? Also do you consider using try catch?

Reply:
I've never seen `Symbol` used but I can add it for completeness